pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.7.0</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.example</groupId>
+	<artifactId>gsoc</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>gsoc</name>
```

Comment:
This needs to be fixed. Please use `io.jenkins.pluginhealth.scoring` for the `groupId` and `plugin-health-scoring` for the `artifactId`.
Same for the name, use `Plugin Health Scoring`.

Reply:
Done, thank you!